<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelAlignDistribute</class>
 <widget class="QWidget" name="PanelAlignDistribute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="SectionContainer" name="sectionAlignment">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title" stdset="0">
      <string>Aligment</string>
     </property>
     <widget class="QWidget" name="layoutAlignment">
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="SectionContainer" name="sectionDistribution">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title" stdset="0">
      <string>Distribution</string>
     </property>
     <widget class="QWidget" name="layoutDistribution">
      <layout class="QVBoxLayout" name="verticalLayout_5"/>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SectionContainer</class>
   <extends>QWidget</extends>
   <header>sectioncontainer.h</header>
   <addpagemethod>addPage</addpagemethod>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
